mpop_kiosk/screensaver.h

```diff
@@ -80,8 +80,8 @@ class AnswersView : public Screensaver
 
 public:
     enum Answers {
+        TheirAnswer,
         UserAnswer,
-        TheirAnswer
```

Comment:
I am not sure about the order of these fields, but overall, the code looks much closer to completion!